sky/shell/platform/mojo/platform_view_mojo.cc

```diff
@@ -162,6 +162,28 @@ void PlatformViewMojo::OnEvent(mojo::EventPtr event,
   callback.Run();
 }
 
+void PlatformViewMojo::SendEvent(pointer::PointerType type, mojo::Event* event, mojo::PointerData* data) {
+  DCHECK(data);
+  pointer::PointerPtr pointer = pointer::Pointer::New();
+  pointer->time_stamp = event->time_stamp;
+  pointer->pointer = data->pointer_id;
+  pointer->type = type;
+  pointer->kind = GetKindFromKind(data->kind);
+  pointer->x = data->x;
+  pointer->y = data->y;
+  pointer->buttons = static_cast<int32_t>(event->flags);
+  pointer->pressure = data->pressure;
+  pointer->radius_major = data->radius_major;
+  pointer->radius_minor = data->radius_minor;
+  pointer->orientation = data->orientation;
+  if (event->action != mojo::EventType::POINTER_UP)
+    pointer_positions_[data->pointer_id] = { data->x, data->y };
+  pointer::PointerPacketPtr packet = pointer::PointerPacket::New();
+  packet->pointers = mojo::Array<pointer::PointerPtr>::New(1);
+  packet->pointers[0] = pointer.Pass();
+  sky_engine_->OnPointerPacket(packet.Pass());
```

Comment:
It's lame that we send the extra moves in a separate packet.  Can we just accumulate a packet and send the whole thing at once?